README.md

```diff
@@ -11,7 +11,47 @@ A developer working on a new microservice should be able to clone this
 repository and build from there. This should lead to greater consistency
 across microservice projects, and cut down on time spent on project setup.
 
-## Running the development server
+## Quick start
+
+There are multiple ways to run this server:
+
+### Docker
+
+1.  Setup [Docker CE using the instructions for your OS](https://docs.docker.com/engine/installation/)
+2.  Build the Docker image, which will execute all the commands in the 
+    [`Dockerfile`](https://github.com/cul-it/arxiv-zero/blob/master/Dockerfile): 
+    `docker build -t arxiv-zero .`
+3.  `docker run -p 8000:8000 --name container_name arxiv-zero` (add a `-d` flag
+    to run in daemon mode)
+3.  Test that the container is working: http://localhost:8000/zero/api/status
+4.  To shut down the container: `docker stop container_name`
+5.  Each time you change a file, you will need to rebuild the Docker image in
```

Comment:
you can also 'docker exec /bin/bash` (to paraphrase a command), but then you need to configure git inside of the container

Reply:
Thanks for the command, Brandon!